### PR TITLE
 [Keyboard] Add nibell/mpad16

### PR DESCRIPTION
**Pull request by Nibell**
## Description

Add nibell/mpad16

## Types of Changes

- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/nibell/mpad16/config.h
Comment (zvecr) [inline keyboards/nibell/mpad16/config.h:1]: Please remove this file as it is not configuring anything.

File: keyboards/nibell/mpad16/rules.mk
Comment (zvecr) [inline keyboards/nibell/mpad16/rules.mk:1]: Please remove this file as it is no longer required.

File: keyboards/nibell/mpad16/info.json
Comment (zvecr) [inline keyboards/nibell/mpad16/info.json:1]: Please rename this file `keyboard.json`

File: keyboards/nibell/mpad16/info.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
+{
+    "manufacturer": "nibell",
+    "keyboard_name": "nibell/mpad16",
+    "maintainer": "Linus Nibell",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
```
Comment (zvecr) [inline keyboards/nibell/mpad16/info.json:10]:
```suggestion
```

File: keyboards/nibell/mpad16/readme.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# nibell/mpad16
+
+![nibell/mpad16](imgur.com image replace me!)
+
+*A short description of the keyboard/project*
+
+* Keyboard Maintainer: [nibell](https://github.com/Linus Nibell)
+* Hardware Supported: *The PCBs, controllers supported*
+* Hardware Availability: *Links to where you can find this hardware*
```
Comment (zvecr) [inline keyboards/nibell/mpad16/readme.md:9]: This content should be filled out

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/nibell/mpad16/rules.mk
Comment (waffle87) [inline keyboards/nibell/mpad16/rules.mk:1]: This file should also be removed.

File: keyboards/nibell/mpad16/config.h
Comment (drashna) [inline keyboards/nibell/mpad16/config.h:1]: This file can be removed.

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by tzarc** [comment]
As of August 26, 2024, `qmk/qmk_firmware` is no longer accepting VIA-enabled keymaps as these have now transitioned to a repository under the VIA team's control.

As you've submitted a PR containing `via` or VIA-enabled keymap(s), this is your notice that they should be removed from this PR. You should now submit a secondary PR to the [VIA QMK Userspace](https://github.com/the-via/qmk_userspace_via) repository with your associated `via` or VIA-enabled keymaps instead.

**Pull request closed without merging**